pkg/nseconfig/config.go

```diff
@@ -18,25 +18,25 @@ type Endpoint struct {
 
 	NseName string `yaml:"nseName"` //TODO temporary in order to be able to run examples
 
-	CNNS *CNNS `yaml:"cnns"`
+	WCM *WCM `yaml:"wcm"`
 
 	VL3 VL3 `yaml:"vl3"`
 }
 
-type CNNS struct {
+type WCM struct {
 	Name               string `yaml:"name"`
 	Address            string `yaml:"address"`
 	AccessToken        string `yaml:"accessToken"`
 	ConnectivityDomain string `yaml:"connectivityDomain"`
 }
 
 type VL3 struct {
-	IPAM        IPAM     `yaml:"ipam"`
+	WCMD        WCMD     `yaml:"wcmd"`
 	Ifname      string   `yaml:"ifName"`
 	NameServers []string `yaml:"nameServers"`
 }
 
-type IPAM struct {
+type WCMD struct {
```

Comment:
I think we should consider splitting IPAM config from service registry config, even if the address is the same. 
Having an IPAM and a ServiceRegistry in the config would be better, and less confusing, then having IPAM as the point for the service registry or the other way around. This can get confusing really fast, and it will also allow the NSE to point to different addresses for IPAM and Registry. Given that this is intended to be open source, this might help in the long run.

So something like this:

```
...
type IPAM struct {
	DefaultPrefixPool string   `yaml:"defaultPrefixPool"`
	PrefixLength      int      `yaml:"prefixLength"`
	Routes            []string `yaml:"routes"`
	ServerAddress     string   `yaml:"serverAddress"`
}

type ServiceRegistry struct {
	ServerAddress     string   `yaml:"serverAddress"`
}

...
```

Reply:
Yeah, I was thinking about that as well but in this diff to limit the changeset I was just thinking to keep it under the same umbrella and call the umbrella "NseControl" or "NseControlServices" or something like that.